test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -378,7 +379,21 @@ protected void processLine(String line, int logLevel) {
       executor.setStreamHandler(streamHandler);
       executor.setWatchdog(watchDog);
 
-      int retValue = executor.execute(cli);
+      // under java11+, ccm selects wrong java version for dse
+      // force it to use java8 by mapping JAVA8_HOME to JAVA_HOME
+      Map<String, String> ccmEnv = null;
+      String javaVersion = System.getProperty("java.version");
+      if (DSE_ENABLEMENT && !javaVersion.startsWith("1.8")) {
```

Comment:
Filed [DSP-23501](https://datastax.jira.com/browse/DSP-23501) to track fixing this on the ccm side

Reply:
Actually for theres a parameter `--jvm_version=8` available on `ccm start` with the version of ccm used with DSE. Once https://github.com/riptano/ccm-private/pull/79 get's merged we can remove this jvm-swapping code and use that approach instead.